### PR TITLE
[zh-CN] fix error document content in the ::-webkit-slider-thumb

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/css/_doublecolon_-webkit-slider-thumb/index.md [hunk 1]
```diff
@@ -1,48 +1,37 @@
...
+## 浏览器兼容性
+
+{{Compat}}
 
 ## 参见
 
-- ::-webkit-slider-runnable-track
-- 其他浏览器内与该功能类似的伪类
+- {{cssxref("::-webkit-slider-runnable-track")}}
+- 其他浏览器与该功能类似的伪类：
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/_doublecolon_-webkit-slider-thumb/index.md:31]:
```suggestion
- 其他浏览器与该功能类似的伪元素：
```

原文是 pseudo-elements

File: files/zh-cn/web/css/_doublecolon_-webkit-slider-thumb/index.md [hunk 2]
```diff
@@ -1,48 +1,37 @@
...
 slug: Web/CSS/::-webkit-slider-thumb
+l10n:
+  sourceCommit: f2175c4c2cf9de232ec9b2e1c395903d53ea24a0
 ---
 
-{{CSSRef}}
+{{CSSRef}}{{SeeCompatTable}}
 
-## 总结
+**`::-webkit-slider-thumb`** CSS [伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)代表 {{HTMLElement("input")}} 元素的滑块按钮，用户可以拖动 `type="range"` 按钮来改变其数值。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/css/_doublecolon_-webkit-slider-thumb/index.md:10]:
```suggestion
**`::-webkit-slider-thumb`** CSS [伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)代表 `type="range"` 的 {{HTMLElement("input")}} 元素的滑块按钮，用户可以拖动按钮来改变其数值。
```

**Review by JasonLamv-t — COMMENTED** [review]
Let's make some imporvements.

File: files/zh-cn/web/css/_doublecolon_-webkit-slider-thumb/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/web/css/_doublecolon_-webkit-slider-thumb/index.md:31]:
```suggestion
- 其他浏览器与之类似的伪元素：
```

File: files/zh-cn/web/css/_doublecolon_-webkit-slider-thumb/index.md [hunk 3]
```diff
@@ -1,48 +1,37 @@
...
-  height: 15px;
+::-webkit-slider-thumb {
+  /* ... */
 }
 ```
 
 ## 规范
 
-这不是任何规范中的一部分，这是 webkit/bink 内核特有的 (chrome/safari)
+不属于任何标准。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/_doublecolon_-webkit-slider-thumb/index.md:22]:
```suggestion
不属于任何规范。
```

File: files/zh-cn/web/css/_doublecolon_-webkit-slider-thumb/index.md [hunk 4]
```diff
@@ -1,48 +1,37 @@
...
+## 浏览器兼容性
+
+{{Compat}}
 
 ## 参见
 
-- ::-webkit-slider-runnable-track
-- 其他浏览器内与该功能类似的伪类
+- {{cssxref("::-webkit-slider-runnable-track")}}
+- 其他浏览器与之类似的伪元素：
```
Comment (jasonren0403) [inline files/zh-cn/web/css/_doublecolon_-webkit-slider-thumb/index.md:31]:
```suggestion
- 其他浏览器使用的与之类似的伪元素：
```

**Review by jasonren0403 — COMMENTED** [review]
与 https://developer.mozilla.org/zh-CN/docs/Web/CSS/::-webkit-slider-runnable-track 对齐

**Review by jasonren0403 — APPROVED** [review]
LGTM 👍

**Comment by T34-active** [comment]
> LGTM 👍

thank you😊

**Pull request merged**